tests/types/test-batch-posts.php

```diff
@@ -122,6 +122,37 @@ public function test_offset_run() {
 		$this->assertEquals( 'finished', $batch_status['status'] );
 	}
 
+	/**
+	 * Test that batch gets run.
+	 */
+	public function test_run_with_destructive_callback() {
```

Comment:
Can we add additional tests to ensure that these work for other batches? Terms and users and sites?